tigerpath/majors_and_certificates/majors/COS-AB.json

```diff
@@ -277,7 +278,8 @@
             "MOL 437/NEU 437",
             "NEU 330",
             "MUS 315",
-            "ECO 326"
+            "ECO 326",
```

Comment:
https://www.cs.princeton.edu/ugrad/becoming-cs-major/requirements

"and SOC 414 count as departmentals."